### ISSUE TITLE
Source js files are not included in npm package for v4

### ISSUE THREAD

**Issue by s7b4** [twbs/bootstrap#18934]
Hi,

When we install bootstrap 3.3.6 with npm, source js directory is included but no more with the v4

with both of the following commands:
`npm install twbs/bootstrap#v4-dev`
or
`npm install bootstrap@4.0.0-alpha.2`

I get this folder content
```
├ node_modules/bootstrap/
├── CHANGELOG.md
├── dist
├── grunt
├── Gruntfile.js
├── LICENSE
├── package.json
├── README.md
├── scss
```
No source js files are present.

Regards
Stéphane

**Comment by petetnt** [comment]
There are UMD-module wrapped (and babelified / es5) files available under `dist/js/umd` which you can use meanwhile (if you can get the dependencies resolved, which can be a pain, I am working on improving the general modularization of Bootstrap v4: https://github.com/twbs/bootstrap/issues/17325#issuecomment-172380793)

**Comment by s7b4** [comment]
Thanks for the answer, i will try this.

**Comment by Avcajaraville** [comment]
I can't see those files. If I get bootstrap v4-dev, I'm only able to see the compiled bootrap file (minified and not minified), but no trace of umd under bootstrap/dist/js.

Im installing bootstrap through npm by having this as dependency on my package.json:
"bootstrap": "twbs/bootstrap#v4-dev"

Any help on how can I have my modules back ?

**Comment by petetnt** [comment]
Hey @Avcajaraville,

there's a recent change (for alpha 3) on how Bootstrap will distribute its source files. See https://github.com/twbs/bootstrap/pull/20072 for more information.

**Comment by rosskevin** [comment]
I don't see es6 sources, umd, or cjs in the alpha.3 npm package.  Am I missing something?

**Comment by cvrebert** [comment]
> I don't see es6 sources,

That's bug #20385.

> umd, or cjs in the alpha.3 npm package.

We resolved in #20072 to no longer provide pre-transpiled AMD, UMD, or CommonJS; see the PR for details.

**Comment by mdo** [comment]
Believe this was resolved in Alpha 4.
